app/main.py

```diff
@@ -24,36 +24,22 @@ def __iadd__(self, other):
         return self
 
     def __mul__(self, other):
-        if isinstance(other, Distance):
-            return Distance(self.km * other.km)
         return Distance(self.km * other)
 
     def __truediv__(self, other):
-        if isinstance(other, Distance):
-            return Distance(round(self.km / other.km, 2))
-        return Distance(round(self.km / other, 2))
+        return Distance(round((self.km / other), 2))
 
     def __lt__(self, other):
-        if isinstance(other, Distance):
-            return self.km < other.km
         return self.km < other
 
     def __gt__(self, other):
-        if isinstance(other, Distance):
-            return self.km > other.km
         return self.km > other
 
     def __eq__(self, other):
-        if isinstance(other, Distance):
-            return self.km == other.km
         return self.km == other
 
     def __le__(self, other):
-        if isinstance(other, Distance):
-            return self.km <= other.km
         return self.km <= other
 
     def __ge__(self, other):
-        if isinstance(other, Distance):
-            return self.km >= other.km
         return self.km >= other
```

Comment:
and only these two methods should be rewritten with previously created magic methods:
`<=` is the same as `not >`

Tag me on your slack channel if it's hard to understand